lessons/operator/03/README.md

```diff
@@ -375,208 +217,446 @@ $ sensuctl configure -n \
+   To get detailed information about an entity, run `sensuctl entity info <entity_id>`.
 
-1. **Export all resources.**
+   ```shell
+   sensuctl entity info learn.sensu.io
+   ```
 
+   **Example Output:**
    ```shell
-   sensuctl dump all
+   === learn.sensu.io
+   Name:                   learn.sensu.io
+   Entity Class:           proxy
+   Subscriptions:          entity:learn.sensu.io, workshop-test
+   Last Seen:              N/A
+   Hostname:               learn.sensu.io
+   OS:                     Workshop
+   Platform:               Sensu Go
+   Platform Family:        Training
+   Platform Version:       6.2.7
+   Auto-Deregistration:    true
+   Deregistration Handler: 
    ```
 
-1. **Export resources for a single namespace.**
+1. **Output Entity Information in JSON Format.**
+
+   Entity information is commonly used in automation scenarios, such as scripts or CI/CD workflows.
+   In those situations it you may wish to output this information in a structured format like JSON. 
 
    ```shell
-   sensuctl dump all --namespace default
-   ```
+   sensuctl entity info learn.sensu.io --format json
+   ```
+
+   **Example Output:**
+   ```json
+   {
+     "entity_class": "proxy",
+     "system": {
+       "hostname": "learn.sensu.io",
+       "os": "Workshop",
+       "platform": "Sensu Go",
+       "platform_family": "Training",
+       "platform_version": "6.2.7",
+       "network": {
+         "interfaces": [
+           {
+             "name": "lo",
+             "mac": "00:00:00:00:00:00",
+             "addresses": [
+               "127.0.0.1/8",
+               "::1/128"
+             ]
+           },
+           {
+             "name": "eth0",
+             "mac": "00:00:00:00:00:00",
+             "addresses": [
+               "10.0.0.1/8"
+             ]
+           }
+         ]
+       },
+       "arch": "arm",
+       "libc_type": "",
+       "vm_system": "",
+       "vm_role": "",
+       "cloud_provider": "",
+       "processes": null
+     },
+     "subscriptions": [
+       "entity:learn.sensu.io",
+       "workshop-test"
+     ],
+     "last_seen": 0,
+     "deregister": true,
+     "deregistration": {},
+     "user": "workshop",
+     "metadata": {
+       "name": "learn.sensu.io",
+       "namespace": "default",
+       "labels": {
+         "app": "workshop"
+       },
+       "created_by": "sensu"
+     },
+     "sensu_agent_version": ""
+   }
+   ```
+
+## Monitoring as Code
+
+The `sensuctl` monitoring-as-code workflow manages resource configurations using plain-text formats like YAML and JSON.
+
+Managing resources follows these basic steps:
+1. Resource configurations are defined in a YAML or JSON file.
+2. The `sensuctl create` command reads the resource configuration from the file, then pushes that configuration to the backend API where the resource is created.
```

Comment:
IMO the `--file` portion of the command is not needed for the description at this point. Again, assuming our reader is a generally knowledgable user of UNIX-like command-line environments, it's understood that "everything is a file" including the STDIN stream, STDOUT stream, as well as any file you might use on disk. This is such an essential part of the UNIX command-line experience that mentioning it, or being explicit about it would just add noise. 

TBH if I had written this CLI tool I would not have required the user to explicitly use the `--file/-f` option and instead would have assumed the first argument to the subcommand create was a filename. If it was not present I would look on STDIN for input. If you want to create without a file or STDIN stream, using only `--whatever` parameters, then that would be the special case. Golden rule of product design is to make the most common case the easiest and least verbose/least steps. Putting a tax on the most common use case in the name of uniformity/consistency just alienates the user.

Reply:
It may make sense to use the verb "send" here vs "push". I think "send" might be more accurate anyway, and "push" is kind of a `git`-ism. Will update to that effect.

Reply:
Not all of our users have Unix experience. We have a lot of Microsoft customers to whom many Unix/Linux concepts are quite foreign. Just FYI.